CONTRIBUTING.md

```diff
@@ -146,6 +146,23 @@ and the rest of contributors.
   [`.github/CODEOWNERS`](./.github/CODEOWNERS) file.
 - Run `make generate-gh-issue-templates` to add your component to the dropdown list in the issue templates.
 
+When submitting a component to the community, consider breaking it down into separate PRs as follows:
+
+* **First PR** should include the overall structure of the new component:
+  * Readme, configuration, and factory implementation usually using the helper
+    factory structs.
+  * This PR is usually trivial to review, so the size limit does not apply to
+    it.
+  * The component should use [`In Development` Stability](https://github.com/open-telemetry/opentelemetry-collector#development) in its README.
+* **Second PR** should include the concrete implementation of the component. If the
+  size of this PR is larger than the recommended size consider splitting it in
+  multiple PRs.
+* **Last PR** should mark the new component as `Alpha` stability and add it to the `cmd/otelcontribcol`
+  binary by updating the `cmd/otelcontribcol/components.go` file. The component must be enabled
+  only after sufficient testing and only when it meets [`Alpha` stability requirements](https://github.com/open-telemetry/opentelemetry-collector#alpha).
+* Once a new component has been added to the executable, please add the component
```

Comment:
Is this needed? I thought we got updates for free there? 

cc @austinlparker

Reply:
Not free (at least that I'm aware of).  I believe this file needs manually updated: https://github.com/open-telemetry/opentelemetry.io/tree/main/data/registry.

Related docs: https://opentelemetry.io/ecosystem/registry/adding/

Reply:
You are right. I saw some new components there that I was confident the author didn't add and thought there was some automatic process involved. Apparently, @svrnm has been doing this:

https://github.com/open-telemetry/opentelemetry.io/pull/2574

Reply:
We have a semi automatic process for adding to the registry. I run it monthly (or bi-monthly) to scan most repos for new components.

Reply:
@jpkrohling Want to keep the line anyways?